pom.xml

```diff
@@ -38,6 +38,11 @@
             <name>jcenter</name>
             <url>https://jcenter.bintray.com</url>
```

Comment:
Please remove this repository as well as you're adding MavenCentral

Reply:
Hmm, I'm not sure why I even added that one, it should work without it as well, I've committed a change to this.

Reply:
I'm unsure it will work without any repository defined

Reply:
It seems like it does actually 👀

Reply:
Yeah, it should work fine if all you need is Maven Central, the Maven Super pom defines that.